docs/Lifecycle.md

```diff
@@ -14,13 +14,13 @@ Incoming Request
                   │
         4**/5** ◀─┴─▶ run Middlewares
                         │
-              4**/5** ◀─┴─▶ Parsing
-                             │
-                       415 ◀─┴─▶ Validation
+              4**/5** ◀─┴─▶ preValidation Hook
+                              │
+                    4**/5** ◀─┴─▶ Parsing
                                    │
-                             400 ◀─┴─▶ preHandler Hook
+                             415 ◀─┴─▶ Validation
                                          │
-                               4**/5** ◀─┴─▶ beforeHandler
+                                   400 ◀─┴─▶ preHandler Hook
                                                │
                                      4**/5** ◀─┴─▶ User Handler
```

Comment:
Can we point out which phases are implemented by hooks?

Reply:
Can you rephrase the question?

Reply:
In that graph, some steps are done by the framework itself, other ones are hooks. Can we use a sign to point the hooks out? for example `*`